datasets/titanic/metadata.json

```diff
@@ -50,10 +73,21 @@
         "description": "Maps Embarkation port initial to labeled values.",
         "key": "#{key}",
         "field": [
-            { "name": "key", "@type": "ml:Field", "dataType": "sc:Text" },
-            { "name": "label", "@type": "ml:Field", "dataType": "sc:Text" },
+            {
+              "name": "key",
+              "description": "Key in the dataset",
```

Comment:
To be coherent with the description above, should it be "C, Q, S or ?"?
I am wondering whether we want to recommend adding a description to this "mapping" RecordSets, which are here to map enumeration values to a semantic meaning and human label.

Reply:
Done.